src/views/CooperationInfo.vue

```diff
@@ -361,6 +564,8 @@ export default defineComponent({
       this.closeCooperationModal();
     },
     editHouseInfo(house: HouseInterface) {
+      console.log(house);
```

Comment:
remove this console.log if you don't need it.

Reply:
removed